includes/helpers/Form_Helper.php

```diff
@@ -5,185 +5,128 @@ class DPLR_Form_helper
 
-		<?php
-		return;
-	}
+	<?php return;
+	endif;
 
     $form = $context['form'];
     $fields = isset($context['fields']) ? $context['fields'] : [];
     $form_class = isset($context['classes']) ? implode(" ", $context['classes']) : "";
     $form_orientation_horizontal = isset($form->settings["form_orientation"]) && $form->settings["form_orientation"] === 'horizontal';
 
     ?>
-    <form class="dplr_form <?php echo $form_class; ?>">
-		<?php
-		if($form_orientation_horizontal):
-		?>
-			<div style="display:flex; gap: 10px;">
-		<?php
-		else:		
-		?>
-			<div>
-		<?php
-		endif;
-		?>
-		<input type="hidden" name="list_id" value="<?php echo $form->list_id; ?>">
-		<input type="hidden" name="form_id" value="<?php echo $form->id; ?>">
-			<?php 
-			foreach ($fields as $field) {
-				$label = isset($field->settings['label']) ? $field->settings['label'] : $field->name;
 
-				if($form_orientation_horizontal &&
-				$field->type !== "permission"):
-				?>
-					<div class="input-field <?php echo isset($field->settings['required']) ? 'required' : ''; ?>"
-					style="display: flex; align-items: center;">
-				<?php
-				else:
-				?>
-					<div class="input-field <?php echo isset($field->settings['required']) ? 'required' : ''; ?>"
-					style="text-align: left;">
-				<?php
-				endif;
-				?>
+	<form class="dplr_form <?php echo $form_class; ?>">	
+
+	<?php
+
+	if($form_orientation_horizontal):?>
+    
+		<div class="container">
+			<input type="hidden" name="list_id" value="<?php echo $form->list_id; ?>">
+			<input type="hidden" name="form_id" value="<?php echo $form->id; ?>">
+			<?php foreach ($fields as $field) :
+				$label = isset($field->settings['label']) ? $field->settings['label'] : $field->name;?>
+				<div class="<?php echo ($field->type !== "permission") ? "flex-item" : ""; ?> input-field <?php echo isset($field->settings['required']) ? 'required' : ''; ?>">
 					<?php if($label!==''): ?>
-						<?php
-						if($form_orientation_horizontal):
-							// if($field->type === 'permission'):
-							// ?>
-								<!-- <label for="<?php //echo $field->name; ?>" >
-							// 		<?php //echo $label; ?>
-							// 	</label> -->
-								<?php 
-							// endif;
-						?>
-							<!-- <label for="<?php // echo $field->name; ?>" style="white-space: nowrap; margin-right: 10px;" >
-								<?php // echo $label; ?>
-							</label> -->
-						<?php
-						else:
-							if($field->type !== 'permission'):
-						?>
+						<?php if($field->type === 'permission'): ?>
 							<label for="<?php echo $field->name; ?>" >
 								<?php echo $label; ?>
 							</label>
-						<?php 
-							endif;
-						endif; 
-					endif;
-					if( ($form_orientation_horizontal && 
-					$field->type !== 'permission') ||
-					(!$form_orientation_horizontal) ):
+						<?php endif; ?>
+						<label for="<?php echo $field->name; ?>" class="horizontal_label" >
+							<?php echo $label; ?>
+						</label>
+					<?php endif;
+					if($field->type !== 'permission'):

```

Comment:
```suggestion
					if($field->type !== 'permission' || (!$form_orientation_horizontal)):

```
Me parece que falta esa validacion, puede ser?